packages/mds-stream/index.ts

```diff
@@ -187,6 +187,15 @@ async function writeTelemetry(telemetry: Telemetry[]) {
   }
 }
 
+const writeTripMetadata = async (metadata: TripMetadata) => {
+  if (env.NATS) {
+    await AgencyStreamNats.writeTripMetadata(metadata)
+  }
+  if (env.KAFKA_HOST) {
+    await AgencyStreamKafka.writeTripMetadata(metadata)
+  }
```

Comment:
These could probably be made concurrent as well.

Reply:
Yeah, we could just append them to an array of promises and then call in a `Promise.all()`.

Reply:
You might even be able to get away with something like:
```ts
await Promise.all([
  env.NATS && AgencyStreamNats.writeTripMetadata(metadata), 
  env.KAFKA_HOST && AgencyStreamKafka.writeTripMetadata(metadata)
])
```

Reply:
Yeah I was thinking that too, but writing out a slightly longer definition of it feels a bit more explicit about what's actually going on.

Reply:
Meh... I don't like let/push so is something like this any better?
```ts
await Promise.all([
  ...(env.NATS ? [AgencyStreamNats.writeTripMetadata(metadata)] : []), 
  ...(env.KAFKA_HOST ? [AgencyStreamKafka.writeTripMetadata(metadata)] : [])
])
```
or something similar using `concat` or, ooh, even a generator function?

Reply:
I like that a bit more, yeah. Will adopt :)